include/hip/spirv_hip_devicelib.hh

```diff
@@ -520,9 +520,22 @@ DEFOPENCL1F_NATIVE(log)
 DEFOPENCL1F_NATIVE(recip)
 DEFOPENCL1F_NATIVE(sqrt)
 DEFOPENCL1F_NATIVE(rsqrt)
+DEFOPENCL2F_NATIVE(powr)
```

Comment:
These should go and replace their respective placeholder under CHIP-SPV/include/hip/devicelib/

I don't see these particular intrinsics defined under CUDA math API, however:

https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__INTRINSIC__SINGLE.html#group__CUDA__MATH__INTRINSIC__SINGLE

Nor do I see them defined under HIP instrisics:

https://github.com/ROCm-Developer-Tools/HIP/blob/develop/docs/markdown/hip_kernel_language.md#floating-point-intrinsics